### ISSUE TITLE
Provide Tutorial inside the app

### ISSUE THREAD

**Issue by jerryn70** [renyuneyun/Easer#160]
Please provide tutorial. Because the UI is confusing for noob users like me. I was totally confused how to set this thing . 
So my request is to provide a tutorial on how to  automate things. 
Or otherwise please make a user friendly ui that everyone can understand (like macrodroid app)

**Comment by renyuneyun** [comment]
Actually there is a (not pretty but useful) [tutorial on the wiki](https://github.com/renyuneyun/Easer/wiki/Usage-example). You may also want to see #62 which is also mentioned in the wiki.

But you are right the UI is not very good, partly because of the bad naming of things. I'm planning to rename some more elements for better (more intuitive) understanding.
A longer plan to improve the UI is to provide a tree-like graph view.

**Comment by jerryn70** [comment]
Not everyone uses github or search internet for tutorials. So tutorial wiki is not useful at all. 
A splash screen tutorial is a good idea . So then the user starts the app for the 1st time he will get more idea about the app

**Comment by renyuneyun** [comment]
Yes, I totally agree a splash tutorial is useful. I also planned to do so.

However, the wiki *is* mentioned in the welcome page/dialog. I know people don't like reading texts, but that page is not a lot of texts.
